rest/server.go

```diff
@@ -541,7 +543,8 @@ func (server *HTTPServer) NewMux() http.Handler {
 	httpHandler = xhttp.NewRequestLogger(httpHandler, server.Name(), serverExtraLogger, time.Millisecond, server.httpConfig.GetPackageLogger())
 
 	// metrics wrapper
-	httpHandler = xhttp.NewRequestMetrics(httpHandler)
+	additionalMetrics := metricsutil.GetAdditionalMetrics(services)
```

Comment:
should not be part of `metricsutil` as it's `rest` specific. need to move it to this package.